server/pike.go

```diff
@@ -18,10 +18,34 @@ func (c *Configuration) pikeNewContact(rc *gin.Context) {
 		return
 	}
 
-	if err = c.pikeActions.AddContact(rc.Request.Context(), receiverPaymail, &requesterContact); err != nil {
+	if err = c.pikeContactActions.AddContact(rc.Request.Context(), receiverPaymail, &requesterContact); err != nil {
 		ErrorResponse(rc, ErrorAddContactRequest, err.Error(), http.StatusExpectationFailed)
 		return
 	}
 
 	rc.Status(http.StatusCreated)
 }
+
+func (c *Configuration) pikeGetPaymentDestinations(rc *gin.Context) {
+	var paymentDestinationRequest paymail.PikePaymentDestinationsRequest
+	err := json.NewDecoder(rc.Request.Body).Decode(&paymentDestinationRequest)
```

Comment:
Don't we have to close the Request.Body in defer? I recall, that it was an issue some time ego.

Reply:
You are right, added defer func which will close the body